### DISCUSSION TITLE
timed overlay only close when the timer has expired

### DISCUSSION THREAD

**Discussion by Nehakrg** [microsoft/fluentui-blazor#1813] [Q&A] [answered]
![image](https://github.com/microsoft/fluentui-blazor/assets/25210450/4b74f8c6-80e8-4ecf-a7e3-0f95b7083311)
I was trying the timed overlay from https://www.fluentui-blazor.net/Overlay and have a question. Is it possible to disable the close on mouse event and ensure that the overlay only closes when the timer has expired? Currently, I am setting visible to false OnClose event.

**Comment by dvoituron** [comment, marked as answer]
Yes, you must add `Dismissable="false"`.

```xml
<FluentOverlay @bind-Visible="@visible" Opacity="0.4" Dismissable="false">
    <FluentProgressRing />
</FluentOverlay>
```

**Reply by Nehakrg** [reply]
Thank you.
